src/transaction.js

```diff
@@ -270,6 +272,91 @@ export default class Transaction {
         this.acc = new Account(network_byte);
     }
 
+    tokenContractDataGen(amount, unity, des) {
```

Comment:
Move all token contract related DataGen methods to new class called `TokenContractDataGenerator`. And rename methods to: 

```
tokenContractDataGen(...) --> createInitData(...)
issueDataGen(...) --> createIssueData(...)
destroyDataGen(...) --> createDestroyData(...)
sendDataGen(...) --> createSendData(...)
...
```

And put payment channel contract & lock contract to other new class too (some methods can be implement later).